README.md

```diff
@@ -140,6 +140,15 @@ To run the tests:
 > vcbuild test
 ```
 
+### Android / Android based devices, aka. Firefox OS
+
+Be sure you have downloaded and extracted [Android NDK](https://developer.android.com/tools/sdk/ndk/index.html) before in a folder. Then run:
+
+```sh
+$ ./android-configure /path/to/you/android-ndk
```

Comment:
Yes, to compile for Android devices you need to use the toolchain in the Android NDK for cross compiling. Inside Android NDK there is an `build/tools/make-standalone-toolchain.sh` script that will create an android-toolchain folder inside io.js folder with the cross binaries to use.